youthful-piety/modernized/03-chapter-02.adoc

```diff
@@ -148,47 +148,47 @@ He will never cast off any who love him so much Inquiry
 being made whether she had thought of Go that morning,
 she replied, "`O! yes, I love Him, and have been thinking of Him with pleasure.`"
 
-The rehearsal of the following lines afforded h much satisfaction:
-
-"`Hope wipes the tear from sorrow`'s eye,
+The rehearsal of the following lines afforded her much satisfaction:
 
+[verse]
+____
+Hope wipes the tear from sorrow`'s eye,
 And faith points upward to the sky:
-
 The promise guides her ardent flight,
-
 And joys, unknown to sense, invite
-
 Those blissful regions to explore,
-
-Where pleasures bloom to fade no more.`"
+Where pleasures bloom to fade no more.
+____
 
 While having her apparel changed,
 she remark "`I shall not need clothes much longer--I shall soon
-be clothed in the robes of the Saviour`'s righteousness and,
+be clothed in the robes of the Saviour`'s righteousness`" and,
 at another time,
 "`I shall soon be in that happy place--soon shall rest my head on my Saviour`'s bosom.
 Why are His chariot wheels so long in coming.`"
 
-It being observed to her that she had lost much sleep in her sickness, "`Yes, said she,
-but
+It being observed to her that she had lost much sleep in her sickness, "`Yes,`" said she,
+"`but
 
-"`In wakeful hours of night
-
-I call my God to mind.`"
+[verse]
+____
+In wakeful hours of night
+I call my God to mind.
+____
```

Comment:
Not sure how to do this with quotation marks and a quote block. It was offset in the original with the quotation marks in the quote block.